src/drivers/Qt/AviRecord.cpp

```diff
@@ -460,7 +460,8 @@ static int encode_frame( unsigned char *inBuf, int width, int height )
 	pic->stride[1] = width/2;
 	pic->stride[2] = width/2;
 
-	ret = x265_encoder_encode( hdl, &nal, &i_nal, pic, &pic_out );
+	x265_picture *pic_out = nullptr;
```

Comment:
Not sure this looks right. Aren't you shadowing

https://github.com/TASEmulators/fceux/blob/f980ec2bc7dc962f6cd76b9ae3131f2eb902c9e7/src/drivers/Qt/AviRecord.cpp#L272

?